lib/htmllint.js

```diff
@@ -39,8 +54,13 @@ module.exports = function( config, done ) {
     var files = config.files.map( path.normalize );
     async.mapSeries( chunkify( files, maxChars ), function( chunk, cb ) {
 
+      // determine proper jarfile invokation and arguments.
+      var args = config.server ? ( config.server.host ? ' -Dnu.validator.client.host=' + config.server.host : '' ) +
```

Comment:
I think these are too many ternaries in one expression. I suggest moving a lot of these into a `cmd()` function that returns the value to pass to `exec()`. Inside that function, use regular `if` statements instead of lots and long ternaries.